lib/jgi_mg_assembly/pipeline_steps/readlength.py

```diff
@@ -6,64 +6,70 @@
 and an output file path. It then writes the generated readlength output to the given file path
 and returns the number of reads. If there's any problems, this just raises a ValueError.
 """
+from __future__ import print_function
```

Comment:
subprocess is imported but unused in this file

Reply:
Fixed